### ISSUE TITLE
"compare against previous version" is failed

### ISSUE THREAD

**Issue by admin9527a** [DonJayamanne/gitHistoryVSCode#302]
> git show b49bf37fb1ea6a833b82146a4ef973938663227c:C:/g:/workspace/discuz/home.php  (completed in 0.078s)
fatal: Path 'C:/g:/workspace/discuz/home.php' does not exist in 'b49bf37fb1ea6a833b82146a4ef973938663227c'

My local code repository in Path g:/ ,not C:/.but when I click "compare against previous version",the outline show errors as above.
How can I fixed it?

**Comment by admin9527a** [comment]
![image](https://user-images.githubusercontent.com/7894866/43177708-20f94af2-8ffc-11e8-8ede-77ba43f6df18.png)

**Comment by androidealp** [comment]
I'm going through the same situation

**Comment by schlupp05** [comment]
for me its the same

**Comment by cspears2002** [comment]
Unfortunately, I'm having the same problem.  For some reason, the extensions appears to be appending a "C:" to the path.

**Comment by GaryB432** [comment]
Don't know if this is the same situation but here's what I'm getting

```
git show 0baf4d13335bea6ddf4ec987a8ec429a78861f68:../../../../../c:/Users/Fred/repo/item.component.html  (completed in 0.048s)
fatal: '../../../../../c:/Users/Fred/repo/item.component.html' is outside repository
```

**Comment by petval** [comment]
I have the same issue

```
git log --pretty=oneline -n1 a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0  (completed in 10.108s)
git show --pretty=oneline --decorate=full --numstat a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0  (completed in 0.135s)
git show --numstat --pretty=oneline -M a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0  (completed in 0.125s)
git show --name-status --pretty=oneline -M a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0  (completed in 0.106s)
git remote get-url origin  (completed in 0.207s)
git remote get-url origin  (completed in 0.29s)
git log --pretty=oneline a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0^1 -n 1 -- C:\d:\git\ZIS-CA-Groupware\SP.Team.App.eSign2.0\system\pscBAF\pscBAF\TimerJobsConfiguration\TJ_eSign2.0_DeleteNotStartedWorkflows.xml  (completed in 0.095s)
fatal: Invalid path 'C:/d:': No such file or directory

git show a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0:C:/d:/git/ZIS-CA-Groupware/SP.Team.App.eSign2.0/system/pscBAF/pscBAF/TimerJobsConfiguration/TJ_eSign2.0_DeleteNotStartedWorkflows.xml  (completed in 0.101s)
fatal: Path 'C:/d:/git/ZIS-CA-Groupware/SP.Team.App.eSign2.0/system/pscBAF/pscBAF/TimerJobsConfiguration/TJ_eSign2.0_DeleteNotStartedWorkflows.xml' does not exist in 'a0b70e5935eeb3fb8ad18ae9b31842b75de2fad0'
```

Version: 1.26.0-insider
Commit: 5fae91d61f12726e149fe5baed342eac2fbf8f88
Date: 2018-07-25T02:04:01.629Z
Electron: 2.0.5
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by yinlinyang** [comment]
for me its the same. It`s can be fixed ?

**Comment by seblee** [comment]
# me too

![image](https://user-images.githubusercontent.com/28698690/43622065-0147349a-970d-11e8-8f11-edbf4ed24a9e.png)

**Comment by errotan** [comment]
Compare against previous version doesn't work for me either. Previously it worked. It prepends ..\.. to path.

> git log --pretty=oneline c0682f102812598d129d3aebd34b4215ba2b2dd3^1 -n 1 -- ..\..\c:\dev\koseven\composer.json  (completed in 0.113s)
fatal: ..\..\c:\dev\koseven\composer.json: '..\..\c:\dev\koseven\composer.json' is outside repository

**Comment by tgiacaman** [comment]
Had same problem, my repo is at D:\GIT and when I tried to use "Compare against previous version" it append "c:/" at the begining of the file path.
![image](https://user-images.githubusercontent.com/7785822/43779305-161b4cfa-9a26-11e8-9a72-fd06498c550e.png)
![image](https://user-images.githubusercontent.com/7785822/43779321-215b142e-9a26-11e8-80f0-4d19af122c8f.png)

**Comment by tonyoldfield** [comment]
Same here.

**Comment by sh5525** [comment]
me too

**Comment by ole1986** [comment]
dup with #307

**Comment by bzuillsmith** [comment]
I get this problem too. Oddly, if I click a option compare a few times, it will eventually work.

**Comment by GaryB432** [comment]
Almost 8 weeks. I’m wondering if this project is being maintained

**Comment by RevFry** [comment]
I believe he mentioned in notes on a pull request that he's been away on holiday but that was 26 days ago. Maybe he's in a European country where they take months off at a time?

Either way, this is a pretty important extension. It would be nice if he had a back up maintainer. :)

**Comment by RevFry** [comment]
Could benefit from Hacktoberfest, too.

**Comment by ole1986** [comment]
This shoudl finally be fixed with version 0.4.3
